File: Taller Tipos de clases/Go/Clase Normal/Libro.go
class Libro
{
    public string Titulo;
    public string Autor;

    public Libro()
    {
        Titulo = "Desconocido";
        Autor = "NA";
    }

    public Libro(string titulo, string autor)
    {
        Titulo = titulo;
        Autor = autor;
    }

    public Libro(string titulo)
    {
        Titulo = titulo;
        Autor = "Autor genérico";
    }

    public Libro(Libro l)
    {
        Titulo = l.Titulo;
        Autor = l.Autor;
    }
}

File: Taller Tipos de clases/JAVA/Clase Normal/Libro.java
class Libro {
    String titulo;
    String autor;

    public Libro() {
        titulo = "Desconocido";
        autor = "NA";
    }

    public Libro(String titulo, String autor) {
        this.titulo = titulo;
        this.autor = autor;
    }

    public Libro(String titulo) {
        this.titulo = titulo;
        this.autor = "Autor genérico";
    }

    public Libro(Libro l) {
        this.titulo = l.titulo;
        this.autor = l.autor;
    }
}
